WordPress/Classes/ViewRelated/Stats/Insights/InsightType.swift

```diff
@@ -21,10 +21,17 @@ enum InsightType: Int, SiteStatsPinnable {
     case allAnnual
 
     // These Insights will be displayed in this order if a site's Insights have not been customized.
-    static let defaultInsights = [InsightType.latestPostSummary,
-                                  .todaysStats,
-                                  .allTimeStats,
-                                  .followersTotals
+    static let defaultInsights: [InsightType] = [.mostPopularTime,
+                                                 .allTimeStats,
+                                                 .todaysStats,
+                                                 .followers,
+                                                 .comments
+    ]
+
+    static let oldDefaultInsights: [InsightType] = [.latestPostSummary,
```

Comment:
Shall we also add a comment note here about why this property exists?

Reply:
for sure!